### PR TITLE
Added References to Operators

### PR DESCRIPTION
**Pull request by smorin**
Added reference to operators like in https://github.com/smorin/elixir/edit/master/lib/elixir/pages/Guards.md

### CODE REVIEW

File: lib/elixir/pages/Operators.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ The following is a table of all the operators that Elixir is capable of parsing,
   * `^^^`
   * `~~~`
 
-The following operators are used by the `Bitwise` module when imported: `&&&`, `^^^`, `<<<`, `>>>`, `|||`, `~~~`. See the documentation for `Bitwise` for more information.
+The following operators are used by the `Bitwise` module when imported: [`&&&`](`Bitwise.&&&/2`), [`^^^`](`Bitwise.^^^/2`), [`<<<`](`Bitwise.<<</2`), [`>>>`](`Bitwise.>>>/2`), [`|||`](`Bitwise.|||/2`), [`~~~`](`Bitwise.~~~/2`). See the documentation for `Bitwise` for more information.
```
Comment (fertapric) [inline lib/elixir/pages/Operators.md]: `~~~` has arity 1. See https://hexdocs.pm/elixir/Bitwise.html#~~~/1

File: lib/elixir/pages/Operators.md [hunk 2]
```diff
@@ -8,40 +8,40 @@ The following is a list of all operators that Elixir is capable of parsing, orde
...
+[`<`](`Kernel.</2`) [`>`](`Kernel.>/2`) [`<=`](`Kernel.<=/2`) [`>=`](`Kernel.>=/2`)      | Left to right
+[`==`](`Kernel.==/2`) [`!=`](`Kernel.!=/2`) [`=~`](`Kernel.=~/2`) [`===`](`Kernel.===/2`) [`!==`](`Kernel.!==/2`) | Left to right
+[`&&`](`Kernel.&&/2`) `&&&` [`and`](`Kernel.and/2`)                                      | Left to right
+[`\|\|`](`Kernel.\|\|/2`) `\|\|\|` [`or`](`Kernel.or/2`)                                 | Left to right
+[`&`](`Kernel.SpecialForms.&/1`)                                                         | Unary
+[`=`](`Kernel.SpecialForms.=/1`)                                                         | Right to left
 `=>`                                                                                     | Right to left
 `\|`                                                                                     | Right to left
-`::`                                                                                     | Right to left
+[`::`](`Kernel.SpecialForms.::/1`)                                                       | Right to left
```
Comment (fertapric) [inline lib/elixir/pages/Operators.md]: `::` has arity 2. See https://hexdocs.pm/elixir/Kernel.SpecialForms.html#%3C%3C%3E%3E/1

File: lib/elixir/pages/Operators.md [hunk 3]
```diff
@@ -8,40 +8,40 @@ The following is a list of all operators that Elixir is capable of parsing, orde
...
-`\|\|` `\|\|\|` `or`                                                                     | Left to right
-`&`                                                                                      | Unary
-`=`                                                                                      | Right to left
+[`@`](`Kernel.@/1`)                                                                      | Unary
+[`.`](`Kernel.SpecialForms../2`)                                                         | Left to right
+[`+`](`Kernel.+/1`) [`-`](`Kernel.-/1`) [`!`](`Kernel.!/1`) [`^`](`Kernel.SpecialForms.^/1`) [`not`](`Kernel.not/1`) `~~~` | Unary
+[`*`](`Kernel.*/2`), [`/`](`Kernel.//2`) .                                               | Left to right
+[`+`](`Kernel.+/2`) [`-`](`Kernel.-/2`)                                                  | Left to right
+[`++`](`Kernel.++/2`) [`--`](`Kernel.--/2`) [`..`](`Kernel.../2`) [`<>`](`Kernel.<>/2`)  | Right to left
+[`in`](`Kernel.in/2`) [`not in`](`Kernel.in/2`)                                          | Left to right
```
Comment (fertapric) [inline lib/elixir/pages/Operators.md]: The second one should be `not_in/2` instead of `in/2`.

**Comment by josevalim** [comment]
Thanks! I removed the links from the operator table because Elixir doesn't assign meaning to its operators. Those operators can do anything. I kept the links though when we are discussing exclusively about the usage of those operators, such as btiwise and comparison.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**